endpoints/telegram.py

```
@@ -12,28 +13,47 @@ def __init__(self, bot: telebot.TeleBot, users_service: UsersService, notificati
 
         @self.bot.message_handler(commands=['balance'])
         def handle_balance_message(message):
-            user_get_balance = users_service.get_balance(message.chat.id)
+            user_get_balance = users_service.get_balance(str(message.chat.id))
             if user_get_balance is None:
                 self.bot.send_message(message.from_user.id, "Please send /start")
             else:
                 self.bot.send_message(message.from_user.id, "Your balance is: " + str(user_get_balance))
-
+        @self.bot.message_handler(commands=['pay'])
+        def handle_balance_message(message):
```

Ruff (0.8.2)

Redefinition of unused `handle_balance_message` from line 34

(F811)